contentcuration/contentcuration/frontend/channelList/views/Channel/StudioMyChannels.vue

```diff
@@ -0,0 +1,533 @@
+          });
+        }
+      },
+      thumbnailSrc(channel) {
+        return channel.thumbnail_encoding && channel.thumbnail_encoding.base64
+          ? channel.thumbnail_encoding.base64
+          : channel.thumbnail_url;
+      },
+    },
+    $trs: {
+      channel: 'New channel',
+      noChannelsFound: 'No channels found',
+      resourceCount: '{count, plural,\n =1 {# resource}\n other {# resources}}',
+      unpublishedText: 'Unpublished',
+      lastPublished: 'Published {last_published}',
+      lastUpdated: 'Updated {updated}',
+      details: 'Details',
+      viewContent: 'View channel on Kolibri',
+      goToWebsite: 'Go to source website',
+      editChannel: 'Edit channel details',
+      copyToken: 'Copy channel token',
+      deleteChannel: 'Delete channel',
+      deleteTitle: 'Delete this channel',
+      removeChannel: 'Remove from channel list',
+      removeBtn: 'Remove',
+      removeTitle: 'Remove from channel list',
+      deletePrompt: 'This channel will be permanently deleted. This cannot be undone.',
+      removePrompt:
+        'You have view-only access to this channel. Confirm that you want to remove it from your list of channels.',
+      channelDeletedSnackbar: 'Channel deleted',
+      channelRemovedSnackbar: 'Channel removed',
+      channelLanguageNotSetIndicator: 'No language set',
+      cancel: 'Cancel',
+    },
+  };
+
+</script>
+
+
+<style lang="scss" scoped>
+
+  .studio-small {
+    .channels-body,
+    .new-channel {
+      width: 100%;
+    }
+  }
+
+  .studio-medium {
+    .channels-body,
+    .new-channel {
+      width: 100%;
+      max-width: 83.33%;
+    }
+  }
+
+  .studio-large {
+    .channels-body,
+    .new-channel {
+      width: 100%;
+      max-width: 50%;
+    }
+  }
+
+  .windowBreakpoint-3 {
+    .channels-body,
+    .new-channel {
+      width: 100%;
+      max-width: 83.33%;
+    }
+  }
+
+  .windowBreakpoint-4 {
+    .channels-body,
+    .new-channel {
+      width: 100%;
+      max-width: 66.66%;
+    }
+  }
+
+  .windowBreakpoint-5 {
+    .channels-body,
+    .new-channel {
+      width: 100%;
+      max-width: 50%;
+    }
+  }
+
+  .my-channels {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-height: calc(100vh - 64px - 48px);
+  }
+
+  .no-channels {
+    padding: 16px 0 0 16px;
+    font-size: 24px;
+  }
+
+  .new-channel {
+    display: flex;
+    justify-content: end;
+    margin-top: 20px;
+  }
+
+  .cards {
+    margin-top: 16px;
+
+    /* check this below class, this should be coming form KDS */
+    ::v-deep .visuallyhidden {
+      position: absolute;
+      width: 1px;
+      height: 1px;
+      padding: 0;
+      margin: -1px;
+      overflow: hidden;
+      clip: rect(0 0 0 0);
+      border: 0;
+    }
+  }
+
+  .cards-title {
+    font-size: 14px;
+  }
+
+  .cards-resource {
+    span:first-child::after {
+      margin: 0 8px;
+      content: '•';
+    }
+  }
+
+  .cards-desc {
+    margin-top: 4px;
```

Comment:
```suggestion
    margin-top: 12px;
```

To make it closer to the current version.

Reply:
addressed it